packages/ethereum-contracts/contracts/superfluid/SuperToken.sol

```diff
@@ -167,7 +165,7 @@ contract SuperToken is
             _approve(
                 holder,
                 spender,
-                _allowances[holder][spender].sub(amount, "SuperToken: transfer amount exceeds allowance"));
+                _allowances[holder][spender] - amount);
```

Comment:
why?

Reply:
It becomes harder for direct etherscan-diagnosis on what is the error.

Reply:
reverted to use safemath